mcs/class/System.Security/Test/System.Security.Cryptography.Pkcs/AlgorithmIdentifierTest.cs

```diff
@@ -125,16 +119,6 @@ public void Oid ()
 			ai.Oid = null;
 			Assert.IsNull (ai.Oid, "Oid-Null");
 		}
-
-		[Test]
-		public void Parameters () 
```

Comment:
why was it removed?

Reply:
There is no `Parameters ` property in CoreFX implementation.